scripts/comparator/m3query.yml

```diff
@@ -0,0 +1,23 @@
+listenAddress:
+  type: "config"
+  value: "0.0.0.0:7201"
+
+backend: grpc
+
+rpc:
+  remotes:
+    - name: "remote"
+      remoteListenAddresses: ["m3comparator:9000"]
```

Comment:
nit: Could we perhaps just dump a one paragraph `README.md` in this directory that explains how all the services connect to each other?

i.e. m3query native query -> remote GRPC fanout to comparator
prometheus -> m3query remote read -> remote GRPC fanout to comparator